### ISSUE TITLE
Font's need to scale on HiDPI screens

### ISSUE THREAD

**Issue by Lucretia** [ovenpasta/adi2#2]
I have bad eyesight, the examples are too small on a 4k screen. Yes you can change the css, but this should be detected and scaled.

**Comment by ovenpasta** [comment]
Please let me know your OS and display settings, it should be handled automatically.
To ovverride use ADI_DISPLAY_SCALE_OVERRIDE=2 ./examples/bin/<example>

**Comment by Lucretia** [comment]
Gentoo. That variable works. Thanks.
